oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -281,4 +282,10 @@
             name = "Document Node Store Compression",
             description = "Select compressor type for collections. 'Snappy' is the default supported compression.")
     String collectionCompressionType() default "snappy";
+
+    @AttributeDefinition(
+            name = "Suspend timeout",
```

Comment:
```suggestion
            name = "Commit Suspend timeout",
```

Reply:
I'll make that change in my next commit.